src/components/TileGrid/SeniorTile.tsx

```diff
@@ -19,16 +19,17 @@ export function SeniorTile({
   setSeniors,
```

Comment:
I don't think `SeniorTile.tsx` is used anywhere right? It might make sense to mark the component as `@deprecated` so someone does not inadvertently use the component